api-client/src/utils/getNavigationDeepLinks.ts

```diff
@@ -0,0 +1,29 @@
+import LocationModel from '../models/LocationModel'
+
+const getNavigationDeepLinks = (
+  location: LocationModel,
+  platform: 'web' | 'android' | 'ios' | string
+): string | null => {
+  if (!location.location || !location.coordinates) {
+    return null
+  }
+  const long = location.coordinates[0]
+  const lat = location.coordinates[1]
+  const android = `geo:${lat},${long}?q=${location.location}`
+
+  switch (platform) {
+    case 'web': {
+      const baseUrl = `https://maps.google.com?q=`
+      const isAndroid = /Android/i.test(navigator.userAgent)
+      return isAndroid ? android : `${baseUrl}${location.location},${lat},${long}`
```

Comment:
Why is no iOS link returned if the user agent matches an iOS device but instead just the gmaps url anyway?